src/Data/Models/Vehicle.cs

```diff
@@ -0,0 +1,11 @@
+namespace Data
+{
+    public class Vehicle
+    {
+        public int Id;
+        public string Model;
+        public string LicensePlate;
+        public string Color;
```

Comment:
Enum for color.

Reply:
dont need, it is fine to be string. 

up to us.. maybe we want shades of colors, enum would be bad for that. Maybe RGB, but it is too much for this practice.